src/stdio.jl

```diff
@@ -45,7 +45,7 @@ function watch_stream(rd::IO, name::AbstractString)
         while !eof(rd) # blocks until something is available
             nb = nb_available(rd)
             if nb > 0
-                write(buf, readbytes(rd, nb))
+                write(buf, @compat(read(rd, nb)))
```

Comment:
You only need the `@compat` macro for syntax changes.   For just function-name changes, it should work fine without `@compat`.